CollAction/Startup.cs

```diff
@@ -108,6 +108,7 @@ public void ConfigureServices(IServiceCollection services)
                 options.MailChimpKey = Configuration["MailChimpKey"];
                 options.MailChimpNewsletterListId = Configuration["MailChimpNewsletterListId"];
             });
+            services.AddApplicationInsightsTelemetry(Configuration);
```

Comment:
Probably better to add it only for production-environments, to prevent dev environments from sending data to this resource.